packages/ui/src/components/select.tsx

```diff
@@ -81,9 +87,11 @@ export type SelectProps<T extends SelectOption<ValueType>> = {
     id?: string;
     options: T[];
     value: T | null;
+    search?: boolean;
     onChange: (value: T) => void;
     renderOption?: (value: T) => ReactElement;
     loading?: boolean;
+    noResultsText?: string;
```

Comment:
To keep consistency with other components needing messages, I'd use a `messages` object prop with key `noText`, wdyt?

Reply:
Yes, definitely.